protos/sources/vega/markets.proto

```diff
@@ -46,6 +46,28 @@ message Future {
   DataSourceSpecToFutureBinding data_source_spec_binding = 5;
 }
 
+// Perps product definition
+message Perps {
+  // Underlying asset for the future.
```

Comment:
for the future as in for future use?

Reply:
Nope as in I copy pasted and left future instead of perps